scripts/python_scripts/generate_network_status.py

```diff
@@ -86,28 +89,90 @@ def fill_status_bar(instance):
 
 
 def generate_value_matrix():
-    network_list = get_networks_list(folder="./networks")
+    network_list = generate_network_list(nova_network_list)
     returning_array = []
     for network in network_list:
         network_data_array = []
         network_data_array.append(
             "[%s](https://explorer.subquery.network/subquery/nova-wallet/nova-wallet-%s)" % (
-                network.title(), network)
+                network.get('name').title(), network.get('name'))
         )
         prod_status, prod_commit, stage_status, stage_comit = generate_progress_status(
-            network)
+            network.get('name'))
+        network_data_array.append(network.get('features'))
         network_data_array.append(stage_status)
         network_data_array.append(prod_status)
         network_data_array.append(stage_comit)
         network_data_array.append(prod_commit)
         returning_array.append(network_data_array)
-        print('%s generated!' % network.title())
+        print('%s generated!' % network.get('name').title())
     returning_array.sort()
     increment = iter(range(1, len(returning_array)+1))
     [network.insert(0, next(increment)) for network in returning_array]
     return returning_array
 
 
+def generate_network_list(url):
+    feature_list = []
+    chains_list = send_http_request(url)
+    available_projects = get_networks_list(folder="./networks")
+    for project in available_projects:
+        with open("./networks/%s/project.yaml" % project, 'r') as stream:
+            project_data = yaml.safe_load(stream)
+        project_genesis = remove_hex_prefix(project_data.get('network').get('genesisHash'))
+        chain = [chain for chain in chains_list if chain.get('chainId') == project_genesis]
+        feature_list.append({
+                "name": project,
+                "genesis": project_genesis,
+                "features": check_features(chain[0] if len(chain) == 1 else None)
+            })
+    return feature_list
+
+def check_features(chain):
+	def has_transfer_history(chain):
+		return True
+
+	def has_orml_or_asset(chain):
+		for asset in chain.get('assets'):
+			if (asset.get('type') in ['orml', 'statemine']):
+				return True
+		return False
+
+	def has_staking_analytics(chain):
+		if (chain.get('assets')[0].get('staking') == 'relaychain'):
+			return True
+		return False
+
+	def has_rewards_history(chain):
+		if (chain.get('assets')[0].get('staking')):
+			return True
+		return False
+
+	dict = {
+		"📚 Operation History" : has_transfer_history,
+		"✨ Multi assets" : has_orml_or_asset,
+		"📈 Staking analytics" : has_staking_analytics,
+		"🥞 Staking rewards": has_rewards_history
+	}
+
+	if (chain == None):
```

Comment:
So if we find an unknown chain we will return the full set of features?
Also, what is the reason to use `next(iter(dict))` instead of `dict` directly?

Reply:
There is if we find an unknown chain we will return just first element of dict. If we just return dict directly it will contain all keys and values instead only one key returned by next(iter()). I will replace it to the more clearly way - list(dict.keys())[0]